src/styles/styles.js

```diff
@@ -3647,6 +3646,18 @@ const styles = {
         willChange: 'transform',
     },
 
+    cartIconContainerPadding: {
```

Comment:
why do we keep `cart` prefix? The dropdown button is a generic component, kindly keep the styles naming in the same way to be consistent